source/simulator/checkpoint_restart.cc

```diff
@@ -43,7 +43,7 @@ namespace aspect
     void move_file (const std::string &old_name,
                     const std::string &new_name)
     {
-      int error = system (("mv " + old_name + " " + new_name).c_str());
+      int error = std::system (("mv " + old_name + " " + new_name).c_str());
```

Comment:
Is this an opportunity to remove the system call altogether? Perhaps something like the suggestions at https://stackoverflow.com/a/48614612/6272561 would be better?

Reply:
I wanted to do that, but we're still on C++14 and don't have access to `std::filesystem`. This will have to wait another year.